### DISCUSSION TITLE
Is there a way to get a bigger input/text box?

### DISCUSSION THREAD

**Discussion by rcdailey** [open-webui/open-webui#4879] [Q&A]
Typically I write very long prompts, especially when code is involved. In these situations, the very small input box is difficult to use directly. It doesn't appear to be resizable, either. What I find myself often doing is using VS Code or Notepad++ to edit my prompt first (I have a large text space to edit in) and then copy/paste that into Open WebUI Prompt text box.

This workflow is a bit cumbersome, so I was hoping someone could help me find a better workflow for this. A rich-text, fancy and resizable text editor would be fantastic. Something akin to Github text boxes used for new issues or pull request descriptions. It's resizable, with mono-spaced font.

But maybe there's a third party tool or something out there that I can use. I'm new to this, so advice is appreciated, assuming this doesn't just need to be a feature request...

**Comment by lmdevv** [comment]
Hey have you found a solution? @rcdailey

**Reply by rcdailey** [reply]
Nope... this likely involves a pull request which I don't have domain expertise or time for.
